crates/core/tedge_mapper/src/c8y/service_monitor.rs

```diff
@@ -0,0 +1,126 @@
+use crate::core::error::ConversionError;
+use c8y_api::smartrest::topic::SMARTREST_PUBLISH_TOPIC;
+use mqtt_channel::Message;
+use mqtt_channel::Topic;
+use serde_json::Value;
+use std::collections::HashMap;
+
+pub fn convert_health_status_message(
+    message: &Message,
+    device_name: String,
+) -> Result<Vec<Message>, ConversionError> {
+    let mut mqtt_messages: Vec<Message> = Vec::new();
+    let topic = message.topic.name.to_owned();
+    let payload: HashMap<String, Value> = serde_json::from_str(message.payload_str()?)?;
+
+    if payload.len() > 1 {
```

Comment:
The `service monitor` message will not be created. This can happen in the case of, a bridge health status message. [tedge/health/mosquitto-c8y-bridge] 1/0.

Reply:
Good point. An option could be to translate 1/0 as up/down. However not sure that this helpful as the point is to transfer this state over the bridge. @reubenmiller what do you think about this case?